mix.exs

```diff
@@ -23,7 +23,7 @@ defmodule TDS.Ecto.Mixfile do
     [
       {:ecto, "~> 1.0.0"},
       {:tds, "~> 0.5.1"},
-      {:poison, only: :test}
+      {:poison, "~> 1.0"}
```

Comment:
Yes, the thing is that I'm using the latest mix version, and if I use a more restrictive dependency specification it generates a conflict with other packages that require poison as optional but not test only. I can remove that change though, it's not important.